docs/product-release-notes/src/main/asciidoc/rn-known-issues-con.adoc

```diff
@@ -8,7 +8,7 @@ This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 * It is possible to morph from a sub-process into a certain task type, but it is not possible to morph back to a sub-process. You should be able to:
```

Comment:
The subject.object seem to be mixed up a bit. And I think you morph a thing *into* not *to* something? Perhaps "It is possible to morph a sub-process into a certain task type, but it is not possible to morph a task type back into a sub-process"?